app/models/worksheet_review.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: worksheet_reviews
+#
+#  id           :bigint           not null, primary key
+#  content      :text
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  user_id      :bigint           not null
+#  worksheet_id :bigint           not null
+#
+# Indexes
+#
+#  index_worksheet_reviews_on_user_id       (user_id)
+#  index_worksheet_reviews_on_worksheet_id  (worksheet_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#  fk_rails_...  (worksheet_id => worksheets.id)
+#
+class WorksheetReview < ApplicationRecord
+  belongs_to :user
+  belongs_to :worksheet
+
+  scope :teacher, ->() {self.worksheet.work_group.classroom.user}
+
+  validates :content, :user, :worksheet, presence: true
```

Comment:
don't need to validate presence of user and worksheet, that's done automatically with the `belongs_to`